azurerm/internal/services/datafactory/data_factory_linked_service_azure_databricks_resource.go

```diff
@@ -483,36 +454,29 @@ func resourceDataFactoryLinkedServiceDatabricksRead(d *schema.ResourceData, meta
 	d.Set("resource_group_name", resourceGroup)
 	d.Set("data_factory_name", dataFactoryName)
 
-	Databricks, ok := resp.Properties.AsAzureDatabricksLinkedService()
-
+	databricks, ok := resp.Properties.AsAzureDatabricksLinkedService()
 	if !ok {
-		return fmt.Errorf("Error classifiying Data Factory Linked Service Databricks %q (Data Factory %q / Resource Group %q): Expected: %q Received: %q", name, dataFactoryName, resourceGroup, datafactory.TypeAzureDatabricks, *resp.Type)
+		return fmt.Errorf("classifiying Data Factory Linked Service Databricks %q (Data Factory %q / Resource Group %q): Expected: %q Received: %q", name, dataFactoryName, resourceGroup, datafactory.TypeAzureDatabricks, *resp.Type)
 	}
 
 	// Check the properties and verify if authentication is set to MSI
-	if props := Databricks.AzureDatabricksLinkedServiceTypeProperties; props != nil {
+	if props := databricks.AzureDatabricksLinkedServiceTypeProperties; props != nil {
+		d.Set("adb_domain", props.Domain)
+
 		if props.Authentication != nil && props.Authentication == "MSI" {
-			authenticationMsi := make(map[string]interface{})
```

Comment:
This is just removing the extra work done to set the msi authentication information into a list